src/main/java/pl/mateusz/zadanie_20/HelloController.java

```diff
@@ -0,0 +1,48 @@
+package pl.mateusz.zadanie_20;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import java.util.List;
+
+@Controller
+public class HelloController {
+
+    private UserRepository userRepository;
+
+    public HelloController(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    User user1 = new User("Mateusz", "Solecki", 25);
+    User user2 = new User("Kamil", "Kowalski", 28);
+    User user3 = new User("Mariola", "Kaczyńska", 36);
+
+    @RequestMapping("/add")
+    public String hello(@RequestParam String firstName, @RequestParam String lastName, @RequestParam Integer age) {
+
```

Comment:
Nie bardzo wiem jak poradzić sobie z poleceniem:
4. W kontrolerze zdefiniuj metody, które obsłużą poniższe adresy:
* Pod adresem /add?imie=Daniel&nazwisko=Abacki&wiek=15 powinno być możliwe dodawanie użytkownika. Upewnij się, że nie dodasz użytkownika z pustym imieniem. Po dodaniu użytkownika powinna wyświetlić się strona success.html, a w przypadku podania pustego imienia strona err.html – oba pliki umieść w folderze static.

Reply:
Częściowo sobie poradziłem, działa tylko jeśli wpiszę np.: /add?imie=&nazwisko=Abacki&wiek=15, a jeśli wpiszę:/add?nazwisko=Abacki&wiek=15 (czyli oprócz imie=) to już pokazuje błąd. Nie wiem czy o to chodziło w zadaniu czy trzeba obsłużyć te dwa warunki czyli gdy: "imie=" czy w ogóle żądanie :"imie=" nie wystąpi?

Reply:
Parametry domyślnie są wymagane, możesz to zmienić dodając atrybut false `@RequestParam(required = false)`